server/command_create.go

```diff
@@ -20,7 +20,7 @@ func getCreateFlagSet() *flag.FlagSet {
 	createFlagSet.String("size", "miniSingleton", "Size of the Mattermost installation e.g. 'miniSingleton' or 'miniHA'")
 	createFlagSet.String("version", "", "Mattermost version to run, e.g. '5.12.4'")
 	createFlagSet.String("affinity", "multitenant", "Whether the installation is isolated in it's own cluster or shares ones. Can be 'isolated' or 'multitenant'")
-	createFlagSet.String("license", "e20", "The enterprise license to use. Can be 'e10' or 'e20'")
+	createFlagSet.String("license", "te", "The enterprise license to use. Can be 'e10', 'e20', or leave blank to choose Team Edition")
```

Comment:
I think the flag description needs to be updated to say `te` instead of blank.

Reply:
Also, not sure that we want to change the default. Thoughts @jwilander?

Reply:
Well, it says blank because it is a default. I misinterpreted this language in the ticket:

>There should also be a "none" or "te" option to create a server with Team Edition.

to mean that "none" ("blank") should be Team.